clmm/utils.py

```diff
@@ -30,6 +30,10 @@ def compute_radial_averages(xvals, yvals, xbins, error_model='std/sqrt_n'):
         Error on the mean y value in each bin. Specified by error_model
     n : array_like
```

Comment:
Minor suggestion: rename to n -> num_objects

Reply:
Done